video-player/src/main/java/io/pravega/example/videoplayer/PravegaReaderVideoPlayer.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package  io.pravega.example.videoplayer;
+
+import io.pravega.example.common.CommonParams;
+import io.pravega.example.common.Utils;
+import io.pravega.example.common.VideoFrame;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.pravega.client.ClientConfig;
+import io.pravega.client.EventStreamClientFactory;
+import io.pravega.client.admin.ReaderGroupManager;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.*;
+import io.pravega.connectors.flink.PravegaConfig;
+import org.bytedeco.javacpp.BytePointer;
+import org.bytedeco.javacv.CanvasFrame;
+import org.bytedeco.javacv.Frame;
+import org.bytedeco.javacv.OpenCVFrameConverter;
+import org.bytedeco.opencv.global.opencv_imgcodecs;
+import org.bytedeco.opencv.opencv_core.Mat;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.util.UUID;
+
+/**
+ * Reads video images a Pravega stream and displays them on the screen.
+ */
+public class PravegaReaderVideoPlayer implements Runnable {
```

Comment:
This seems to be a duplicate of VideoPlayer except that this uses JavaSerializer. JavaSerializer should not be used. You should probably delete this file.